tools/challenge-helper-scripts/create-project.ts

```diff
@@ -1,5 +1,5 @@
-import fs from 'fs/promises';
-import { existsSync } from 'fs';
```

Comment:
Just wanted to be sure, is this change intentional?

Reply:
Yes @huyenltnguyen it was indeed intentional.. there was an eslint error so to resolve it I had to change this code..

Reply:
Could you help share the error and perhaps your Node version as well?

I find it odd that these ESLint plugins would touch non-test files.

Reply:
Oh this is weird, I remember getting an error now I am not getting.. I will revert the same..